### ISSUE TITLE
Ghost with multiple subdirectories

### ISSUE THREAD

**Issue by okrivtsov** [TryGhost/Ghost#6700]
Hello! 

I installed Ghost at two subfolders (at one domain).

```
www
|
|---/life
|---/data
```

**Issue:**

I open tab and try login domain.com/**life**/ghost. OK! 
I open new tab and try login domain.com/**data**/ghost. Not ok! I automatically logout on first tab and can't login on second tab. If i close first tab i can login on domain.com/**data**/ghost. Why?

**My nginx config:**

```
server {
    listen 80 default_server;
    listen [::]:80 default_server ipv6only=on;

    server_name domain.com; # Replace with your domain

    root /usr/share/nginx/html;
    index index.html index.htm;

    client_max_body_size 10G;

	location / {
	}
	
	
    location ^~ /data {
        proxy_pass http://localhost:2369;
		proxy_set_header X-Real-IP  $remote_addr;
        proxy_set_header X-Forwarded-For $remote_addr;
        proxy_set_header Host $host;
    } 

    location ^~ /life {
        proxy_pass http://localhost:2368;
		proxy_set_header X-Real-IP  $remote_addr;
        proxy_set_header X-Forwarded-For $remote_addr;
        proxy_set_header Host $host;
    } 
}

```

**Ghost config 1:**

```
var path = require('path'),
    config;

config = {
    // ### Production
    // When running Ghost in the wild, use the production environment.
    // Configure your URL and mail settings here
    production: {
        url: 'http://domain.com/life/',
        mail: {},
        database: {
            client: 'sqlite3',
            connection: {
                filename: path.join(__dirname, '/content/data/ghost.db')
            },
            debug: false
        },

        server: {
            host: '127.0.0.1',
            port: '2368'
        }
    },
```

Ghost config 2:

```
var path = require('path'),
    config;

config = {
    // ### Production
    // When running Ghost in the wild, use the production environment.
    // Configure your URL and mail settings here
    production: {
        url: 'http://domain.com/data/',
        mail: {},
        database: {
            client: 'sqlite3',
            connection: {
                filename: path.join(__dirname, '/content/data/ghost.db')
            },
            debug: false
        },

        server: {
            host: '127.0.0.1',
            port: '2369'
        }
    },
```

What have I done wrong? Or is it Ghost error?

**Comment by kevinansfield** [comment]
Hi @okrivtsov, which version of ghost are you using? There was a similar bug to this that was fixed in 0.7.6 although I would recommend updating to the latest 0.7.8 if you are on an older version.

**Comment by okrivtsov** [comment]
@kevinansfield after update to 0.7.8  all is ok! 
Thx!
